src/OrchardCore.Themes/TheAdmin/Views/Message.cshtml

```diff
@@ -1,4 +1,7 @@
-<div class="alert alert-warning alert-dismissible fade show" role="alert">
+@{
+    string type = Model.Type.ToString().ToLowerInvariant();
+}
+<div class="alert alert-dismissible fade show message-@type" role="alert">
```

Comment:
Why not using `Model.Type.ToString().ToLowerInvariant();` directly instead?

Reply:
Yes probably could be @Skrypt I did it with `string type` as it is dynamic, and a bit more explicit. will check tomorrow

Reply:
Well I tried with my website and it works fine. You cast it as a String when you do .ToString() anyways 😄